src/Model/Tags/Tag.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
You aren't using the Tag type so you don't need it.

Reply:
The Tag type is used in `TagsService.php`.